presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -308,9 +308,14 @@ primaryExpression
     | '(' expression ')'                                                                  #parenthesizedExpression
     ;
 
+string
+    : STRING
+    | ESCAPE_INTRODUCER STRING
```

Comment:
I think you'll need to do something like this, so that the Unicode string literal is a single token:
```antlr
string
    : BASIC_STRING    #basicString
    | UNICODE_STRING  #unicodeString
    ;

UNICODE_STRING
    : ESCAPE_INTRODUCER STRING

BASIC_STRING:
    : STRING
    ;

fragment STRING
    : '\'' ( ~'\'' | '\'\'' )* '\''
    ;